tmpl/package.json

```diff
@@ -15,7 +15,7 @@
       "make_targets": {
         "win32": ["squirrel"],
         "darwin": ["zip"],
-        "linux": ["deb", "rpm"]
+        "linux": ["deb", "rpm", "flatpak"]
```

Comment:
Be aware that this would make `flatpak` a default for all users of `electron-forge`.  Is this what we want?